### ISSUE TITLE
macOS conda-forge failed assertion: `FE_INVALID raised`

### ISSUE THREAD

**Issue by mwtoews** [libgeos/geos#930]
Over at conda-forge, the latest build for 3.12.0 with https://github.com/conda-forge/geos-feedstock/pull/70 has one test failure on macOS 12.6.6 with Clang 15.0.7.

Relevant excerpts from the [log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=735425&view=logs&jobId=58ac6aab-c4bc-5de2-2894-98e408cc8ec9&j=58ac6aab-c4bc-5de2-2894-98e408cc8ec9&t=933f325c-924e-533d-4d95-e93b5843ce8b) are captured here:
```
===============================
  GEOS Unit Test Suite
===============================

geos::geom::Envelope: .....[6=F]..............

---> group: geos::geom::Envelope, test: test<6>
     problem: assertion failed
     failed assertion: `FE_INVALID raised`
```
See https://github.com/libgeos/geos/blob/3.12.0/tests/unit/geom/EnvelopeTest.cpp#L219-L256 for code reference.

This is probably related to #791

**Comment by pramsey** [comment]
This seems to be limited to x86_64, I am not seeing it on aarm64.

**Comment by mwtoews** [comment]
That's correct, no issue with aarm64 on conda-forge too. The issue with x86_64 was skipped for the [build recipe](https://github.com/regro-cf-autotick-bot/geos-feedstock/blob/2f8bbe2a1532960c9bf6c0e6ae66a52cd8270866/recipe/build.sh), unless a better solution is found. It's possibly compiler-dependant too. And as you know tracing `FE_INVALID` issues is not easy. Consider this a low priority.
